src/libs/ReportUtils.js

```diff
@@ -310,19 +310,28 @@ function hasExpensifyEmails(emails) {
     return _.intersection(emails, CONST.EXPENSIFY_EMAILS).length > 0;
 }
 
+/**
+ * Returns an array after removing expensify emails
+ * @param {Array} emails
+ * @returns {Array}
+ */
+function removeExpensifyEmails(emails) {
+    return _.difference(emails, CONST.EXPENSIFY_EMAILS);
+}
+
 /**
  * Whether the time row should be shown for a report.
  * @param {Array<Object>} personalDetails
  * @param {Object} report
  * @return {Boolean}
  */
 function canShowReportRecipientLocalTime(personalDetails, report) {
-    const reportParticipants = lodashGet(report, 'participants', []);
-    const hasMultipleParticipants = reportParticipants.length > 1;
-    const reportRecipient = personalDetails[reportParticipants[0]];
+    const reportParticipants = _.without(lodashGet(report, 'participants', []), sessionEmail);
+    const participantsWithoutExpensifyEmails = removeExpensifyEmails(reportParticipants);
+    const hasMultipleParticipants = participantsWithoutExpensifyEmails.length > 1;
+    const reportRecipient = personalDetails[participantsWithoutExpensifyEmails[0]];
```

Comment:
participantsWithoutExpensifyEmails could be null, let's make sure to handle that case

Reply:
At line 335 we check if `reportRecipient` is there. If it's not we return false.